### DISCUSSION TITLE
Как получить значок YOLO

### DISCUSSION THREAD

**Discussion by AlexanderMarobe** [community/community#19434] [Other Feature Feedback, Questions, & Ideas] [answered]
Как получить значок YOLO

**Comment by RobertGoodman08** [comment, marked as answer]
чтобы получить значок YOLO, вам необходимо объединить PR напрямую с общедоступным репозиторием кода без проверки.

**Comment by Zako77xx** [comment]
git-for-windows:rebase-to-v2.35.3

**Comment by ghostinhershell** [comment]
Hi @AlexanderMarobe, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
